skbio/stats/composition.py

```diff
@@ -708,6 +715,10 @@ def ancom(table, grouping,
 
         `"reject"` indicates if feature is significantly different or not.
 
+    pd.DataFrame
+        A table of features and their percentile abundances in each group. If
+        ``percentiles = []``, this will be an empty data frame.
```

Comment:
Can you also describe here what the overall structure of the data frame is? I can't tell from the doctest examples below because only pieces of the data frame are displayed. Is it a data frame with features as index and MultiIndex on columns, where groups are nested under percentiles?